Osiris/GUI.cpp

```diff
@@ -1175,16 +1178,46 @@ void GUI::renderConfigWindow(bool contentOnly) noexcept
             ImGui::EndPopup();
         }
         if (currentConfig != -1) {
-            if (ImGui::Button("Load selected", { 100.0f, 25.0f })) {
-                config.load(currentConfig);
-                updateColors();
-                SkinChanger::scheduleHudUpdate();
-                Misc::updateClanTag(true);
+            if (ImGui::Button("Load selected", { 100.0f, 25.0f }))
+                ImGui::OpenPopup("Config to load");
+            if (ImGui::BeginPopup("Config to load")) {
+                static constexpr const char* choices[]{"Confirm", "Cancel"};
```

Comment:
This should be made a separate function to avoid duplicate code. Also no need for loop for only two possible choices.